### PR TITLE
done with modal designing

### PR DESCRIPTION
**Pull request by ravi1225**

### CODE REVIEW

File: todo.js [hunk 1]
```diff
@@ -57,24 +54,46 @@ export default function Todo() {
...
+    },
+
+    header_text: {
+        fontSize: 20,
+        fontWeight: 'bold',
+        paddingBottom: 10,
+    },
+
+    text_style: {
+        color: 'blue',
```
Comment (priyarajputexc) [inline todo.js]: create a new file for constants like color blue. so that you don't repeat strings.

File: todo.js [hunk 2]
```diff
@@ -57,24 +54,46 @@ export default function Todo() {
...
+        paddingRight: 10,
+    },
+
+    header_text: {
+        fontSize: 20,
+        fontWeight: 'bold',
+        paddingBottom: 10,
+    },
+
+    text_style: {
```
Comment (priyarajputexc) [inline todo.js]: use camel case instead of snake case.
example - 
`textStyle` not `text_style`

Comment (ravi1225) [reply todo.js]: Ok

File: src/components/AddTodo.js [hunk 3]
```diff
@@ -1,51 +1,101 @@
...
+        fontSize: 20,
+        fontWeight: 'bold',
+        paddingBottom: 10,
+    },
+
+    textStyle: {
+        color: 'blue',
+        fontSize: 20,
+    }
   });
```
Comment (priyarajputexc) [inline src/components/AddTodo.js]: add empty line at the end of file

File: src/components/Color.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const Colors = {
+    green: '#03fc7b',
+    red: '#F24444',
+    gold: '#D9C57E',
+  };
```
Comment (priyarajputexc) [inline src/components/Color.js]: use colors you are using.
what is this?
that was just an example

File: todo.js [hunk 5]
```diff
@@ -1,100 +1,90 @@
 import React, { useState } from 'react';
-import { FlatList, StyleSheet, Text, TextInput, View, Modal, TouchableOpacity } from 'react-native';
+import { FlatList, StyleSheet, Text, View, Alert, Checkbox, TouchableOpacity} from 'react-native';
```
Comment (priyarajputexc) [inline todo.js]: checkbox is deprecated. don't use it

File: App.js [hunk 6]
```diff
@@ -23,8 +24,14 @@ export default function App() {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: Colors.white,
     marginTop: Constants.statusBarHeight
     
   },
 });
+
```
Comment (priyarajputexc) [inline App.js]: only one empty line.
not 5-6

File: heading.js [hunk 7]
```diff
@@ -53,4 +55,10 @@ const styles = StyleSheet.create({
         
     }
 
-})
\ No newline at end of file
+})
+
```
Comment (priyarajputexc) [inline heading.js]: same here

File: src/components/AddTodo.js [hunk 8]
```diff
@@ -72,6 +66,15 @@ const styles = StyleSheet.create({
         paddingRight: 10,
         minWidth: 350,
         borderColor: Colors.lightgray,
+        backgroundColor: Colors.white,
+        shadowColor: "#000",
```
Comment (priyarajputexc) [inline src/components/AddTodo.js]: use constant colors everywhere. i can not point all the mistakes

File: todo.js (hunk 5 above)
Comment (priyarajputexc) [reply todo.js]: not done yet

File: src/components/AddTodo.js (hunk 8 above)
Comment (priyarajputexc) [reply src/components/AddTodo.js]: not fixed yet. check colors EVERYWHERE

File: src/components/Colors.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+export const Colors = {
+    green: '#03fc7b',
+    red: '#F24444',
+    gold: '#D9C57E',
+    lightgray: '#D3D3D3',
+    black: '#000000',
+    blue: '#0000FF',
+    CornflowerBlue: '#6495ED',
+    white: '#FFFFFF'
+  };
```
Comment (priyarajputexc) [inline src/components/Colors.js]: unused colors.
do you need these green, red gold colors?
you just copy pasted my example

File: todo.js [hunk 10]
```diff
@@ -1,81 +1,88 @@
...
-        //         )}          
-        //    />        
-        // </View>
+    const [todo, setTodo] = useState([
+        {
+           text: 'abc',  key: 1, check:false
+        },
+        {
+           text: 'xyz',  key: 2, check:false
+        },
```
Comment (priyarajputexc) [inline todo.js]: unformatted code

File: todo.js [hunk 11]
```diff
@@ -1,81 +1,88 @@
...
+        paddingRight: 10,
+        marginLeft: 10,
+        marginRight: 10,
+        marginBottom: 0.5,
+        fontSize: 25,
+        borderBottomWidth: 0.5,
+        minWidth:325,
+        borderBottomColor: Colors.lightgray,
+    },
+});
```
Comment (priyarajputexc) [inline todo.js:99]: format all the files

**Pull request merged**